internal/pkg/policy/secret.go

```diff
@@ -177,13 +177,17 @@ func ProcessOutputSecret(ctx context.Context, output smap.Map, bulker bulk.Bulk)
 }
 
 // replaceStringRef replaces values matching a secret ref regex, e.g. $co.elastic.secret{<secret ref>} -> <secret value>
+// and does this for multiple matches
 func replaceStringRef(ref string, secretValues map[string]string) string {
 	matches := secretRegex.FindStringSubmatch(ref)
-	if len(matches) > 1 {
+	for len(matches) > 1 {
```

Comment:
Would `ReplaceAll` be any simpler here? 

https://pkg.go.dev/strings#ReplaceAll

Reply:
I thought about that, but then we would have to rewrite the `FindStringSubmatch` logic too, to find all secret refs in one regex (currently it only finds one).